about.html

```diff
@@ -7,67 +7,83 @@
   <title>Sprint Challenge - About</title>
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+<link rel="stylesheet" href="css/about.css">
+
 
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
-
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    Let's Work Together
+  <header>
+    <div class="bucket">
+    <nav>
+        <div class="head-logo">
+            <img src="img/lambda-black.png" alt="lambda">
+        </div>
+        <div class="links">
+            <a class="home" href="index.html">Home</a>
+            <a class="about" href="about.html">About</a>
+            <a class="products" href="#">Products</a>
+            <a class="blog" href="#">Blog</a>
+            <a class="contact" href="#">Contact</a>
+        </div>
+    </nav>
+    <div class="pic">
+        <img src="img/jumbo-about.png" alt="">
+    </div>
+    </div>
+</header>
+    <div class="container">
+      <div class="about-info">
+    <h1>About</h1>
+    <p>Leverage agile frameworks to provide a robust
+      synopsis for high level overviews. Iterative approaches
+      to corporate strategy foster collaborative thinking to further
+      the overall value proposition. Organically grow the holistic world
+      view of disruptive innovation via workplace diversity and empowerment.</p>
+      </div>
+
```

Comment:
Good job targeting specific areas of the content and applying flexbox to get the layout to be like the design page.